src/Common/ProfileEvents.cpp

```diff
@@ -341,6 +341,7 @@
     \
     M(WaitMarksLoadMicroseconds, "Time spent loading marks", ValueType::Microseconds) \
     M(BackgroundLoadingMarksTasks, "Number of background tasks for loading marks", ValueType::Number) \
+    M(SynchronousMarksTasks, "Number of times marks was loaded synchronously because they were presented in cache already", ValueType::Number) \
```

Comment:
```suggestion
    M(SynchronousMarksTasks, "Number of times marks were loaded synchronously because they were already present in the cache.", ValueType::Number) \
```